app/scenes/channel_info/channel_info.js

```diff
@@ -169,23 +171,27 @@ class ChannelInfo extends PureComponent {
                     </View>
                     <ChannelInfoRow
                         action={this.props.actions.goToChannelMembers}
-                        defaultMessage='Manage Members'
+                        defaultMessage={isAdmin ? 'Manage Members' : 'View Members'}
```

Comment:
Actually I didn't I thought it was better to just keep what we have, cause changing that will require a change in the webapp and I'm 1/5 on the wording